pkg/ioutils/bytespipe_test.go

```diff
@@ -124,6 +124,7 @@ func TestBytesPipeWriteRandomChunks(t *testing.T) {
 }
 
 func BenchmarkBytesPipeWrite(b *testing.B) {
+	testData := []byte("pretty short line, because why not?")
```

Comment:
quoted "before" bench numbers are after this change, otherwise allocs are at 1018/1010 and don't really reflect very well the actual performance of this fn.